.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request professional 😊